src/sentry/migrations/0429_backfill_org_member_id_organizationmembermapping.py

```diff
@@ -0,0 +1,54 @@
+# Generated by Django 2.2.28 on 2023-04-26 19:54
+
+from django.db import migrations
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def backfill_org_member_id_organizationmembermapping(apps, schema_editor):
+    OrganizationMemberMapping = apps.get_model("sentry", "OrganizationMemberMapping")
+    OrganizationMember = apps.get_model("sentry", "OrganizationMember")
+    for org_member_mapping in RangeQuerySetWrapperWithProgressBar(
+        OrganizationMemberMapping.objects.all()
+    ):
+        if org_member_mapping.organizationmember_id is None:
```

Comment:
Should this be a condition for the query? The column has an index and we don't need to update records that already have a value set.

Reply:
The index is unlikely to be used, since the range wrapper sorts/filters by id, but if this table is small enough it might not be a huge deal either way

Reply:
There are currently ~8000 rows in this table. That seems doable in the deploy window.